src/net_processing.cpp

```diff
@@ -725,6 +725,17 @@ void Misbehaving(NodeId pnode, int howmuch)
 // blockchain -> download logic notification
 //
 
+// To prevent fingerprinting attacks, only send blocks/headers outside of the
+// active chain if they are no more than a month older (both in time, and in
+// best equivalent proof of work) than the best header chain we know about.
+static bool OrphanBlockRequestAllowed(const CBlockIndex* pindex, const Consensus::Params& consensusParams)
```

Comment:
snake_case argument names

Reply:
`consensusParams` is camelCase everywhere else in the file, so I think it's better to leave as is. Feel free to replace everywhere in a separate PR.

Reply:
Style-only PRs are highly discouraged, developer guide says to use the style in all new code regardless of style around it, because eventually it will all be replaced

Reply:
Just `params`?

Reply:
@promag The variable is referred to as `consensusParams` everywhere else, probably to differentiate from an instance of `CChainParams`.